### PR TITLE
Add references to fixed bugs

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Two bugs were left out of the release notes by mistake

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- https://github.com/pingcap/tidb/pull/28528
- https://github.com/pingcap/tidb/pull/27216

### CODE REVIEW

**Review by morgo — APPROVED** [review]

**Review by kennytm — APPROVED** [review]

File: releases/release-5.3.0.md [hunk 1]
```diff
@@ -348,6 +348,8 @@ To learn more about telemetry and how to disable this behavior, refer to [Teleme
     - Fix the issue of `data too long` error when dumping statistics from the table with `new collation` data [#27024](https://github.com/pingcap/tidb/issues/27024)
     - Fix the issue that the retried transactions' statements are not included in `TIDB_TRX` [#28670](https://github.com/pingcap/tidb/pull/28670)
     - Fix the wrong default value of the `plugin_dir` configuration [28084](https://github.com/pingcap/tidb/issues/28084)
+    - `CONVERT_TZ()` now supports mixing named timezones and UTC offsets [28528](https://github.com/pingcap/tidb/pull/28528)
```
Comment (TomShawn) [inline releases/release-5.3.0.md:351]:
```suggestion
    - Fix the issue that the `CONVERT_TZ` function returns invalid time format [#8311](https://github.com/pingcap/tidb/issues/8311)
```

File: releases/release-5.3.0.md [hunk 2]
```diff
@@ -348,6 +348,8 @@ To learn more about telemetry and how to disable this behavior, refer to [Teleme
...
+    - CREATE SCHEMA now uses `character_set_server` and `collation_server` for new schemas if none are provided as part of the statement [27216](https://github.com/pingcap/tidb/pull/27216)
```
Comment (TomShawn) [inline releases/release-5.3.0.md:352]:
```suggestion
    - Fix the issue that `collation_server` does not take effect when no collation is specified on the `CREATE SCHEMA` statement [#27214](https://github.com/pingcap/tidb/issues/27214)
```

**Review by TomShawn — COMMENTED** [review]
The bug fix notes should describe the bug instead of the work done to fix the bug.

File: releases/release-5.3.0.md (hunk 1 above)
Comment (dveeden) [reply releases/release-5.3.0.md:351]:
```suggestion
    - Fix the issue that `CONVERT_TZ` returned `NULL` when given a named timezone and an UTC offset [#8311](https://github.com/pingcap/tidb/issues/8311)
```

**Comment by dveeden** [comment]
PTAL @TomShawn

File: releases/release-5.3.0.md [hunk 3]
```diff
@@ -348,6 +348,8 @@ Starting from TiCDC v5.3.0, the cyclic replication feature between TiDB clusters
     - Fix the issue of `data too long` error when dumping statistics from the table with `new collation` data [#27024](https://github.com/pingcap/tidb/issues/27024)
     - Fix the issue that the retried transactions' statements are not included in `TIDB_TRX` [#28670](https://github.com/pingcap/tidb/pull/28670)
     - Fix the wrong default value of the `plugin_dir` configuration [#28084](https://github.com/pingcap/tidb/issues/28084)
+    - `CONVERT_TZ()` now supports mixing named timezones and UTC offsets [28528](https://github.com/pingcap/tidb/pull/28528)
```
Comment (TomShawn) [inline releases/release-5.3.0.md:351]:
```suggestion
    - - Fix the issue that the `CONVERT_TZ` function returns `NULL` when it is given a named timezone and a UTC offset [#8311](https://github.com/pingcap/tidb/issues/8311)
```

File: releases/release-5.3.0.md (hunk 2 above)
Comment (TomShawn) [reply releases/release-5.3.0.md:352]: @dveeden Could you please rephrase the note as "Fix + issue" or consider my comment above?

File: releases/release-5.3.0.md [hunk 4]
```diff
@@ -348,6 +348,8 @@ Starting from TiCDC v5.3.0, the cyclic replication feature between TiDB clusters
     - Fix the issue of `data too long` error when dumping statistics from the table with `new collation` data [#27024](https://github.com/pingcap/tidb/issues/27024)
     - Fix the issue that the retried transactions' statements are not included in `TIDB_TRX` [#28670](https://github.com/pingcap/tidb/pull/28670)
     - Fix the wrong default value of the `plugin_dir` configuration [#28084](https://github.com/pingcap/tidb/issues/28084)
+    - Fix the issue that the `CONVERT_TZ` function returns `NULL` when it is given a named timezone and a UTC offset [#8311](https://github.com/pingcap/tidb/issues/8311)
+    - Fix `CREATE SCHEMA` not using the character set specied by `character_set_server` and `collation_server` for new schemas if none are provided as part of the statement [27216](https://github.com/pingcap/tidb/pull/27216)
```
Comment (TomShawn) [inline releases/release-5.3.0.md:352]: What does "none" refer to?
```suggestion
    - Fix the issue that `CREATE SCHEMA` does not use the character set specified by `character_set_server` and `collation_server` for new schemas if none are provided as part of the statement [27216](https://github.com/pingcap/tidb/pull/27216)
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: releases/release-5.3.0.md [hunk 5]
```diff
@@ -348,6 +348,8 @@ Starting from TiCDC v5.3.0, the cyclic replication feature between TiDB clusters
     - Fix the issue of `data too long` error when dumping statistics from the table with `new collation` data [#27024](https://github.com/pingcap/tidb/issues/27024)
     - Fix the issue that the retried transactions' statements are not included in `TIDB_TRX` [#28670](https://github.com/pingcap/tidb/pull/28670)
     - Fix the wrong default value of the `plugin_dir` configuration [#28084](https://github.com/pingcap/tidb/issues/28084)
+    - Fix the issue that the `CONVERT_TZ` function returns `NULL` when it is given a named timezone and a UTC offset [#8311](https://github.com/pingcap/tidb/issues/8311)
+    - Fix the issue that `CREATE SCHEMA` does not use the character set specified by `character_set_server` and `collation_server` for new schemas if none are provided as part of the statement [27216](https://github.com/pingcap/tidb/pull/27216)
```
Comment (shichun-0415) [inline releases/release-5.3.0.md:352]: [27216](https://github.com/pingcap/tidb/pull/27216) > [#27214](https://github.com/pingcap/tidb/issues/27214)

Comment (dveeden) [reply releases/release-5.3.0.md:352]: @shichun-0415 you're saying the link is wrong? I think that needs a new PR if that's the case as this one has been merged already.

Comment (TomShawn) [reply releases/release-5.3.0.md:352]: Each release note should have an issue appended instead of a PR. Also, the numbers are prefixed with `#`.

**Pull request merged**